hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -326,6 +328,16 @@ private boolean onDeltaSyncShutdown(boolean error) {
     @Parameter(names = {"--checkpoint"}, description = "Resume Delta Streamer from this checkpoint.")
     public String checkpoint = null;
 
+    /**
+     * 1. string: topicName,partition number 0:offset value,partition number 1:offset value
```

Comment:
this format is specific to kafka. lets call it out. other sources could have checkpoint differently.